runtime/compiler/env/J9VMMethodEnv.cpp

```diff
@@ -74,3 +77,39 @@ J9::VMMethodEnv::bytecodeSize(TR_OpaqueMethodBlock *method)
                      J9_BYTECODE_START_FROM_ROM_METHOD(romMethod));
    }
 
+
+void
+J9::VMMethodEnv::tokenizeSignature(
+      const char * methodSignature,
+      const char * &methodClass,
+      uint32_t     &methodClassLen,
+      const char * &methodName,
+      uint32_t     &methodNameLen,
+      const char * &typeSignature,
+      uint32_t     &typeSignatureLen)
+   {
+
+   if (methodSignature[0] == '/') // omr method pattern
+      {
+      methodClass = methodSignature;
+      typeSignature = strchr(methodSignature, ':');
+      methodClassLen = typeSignature - methodClass;
+      typeSignature++;
+      methodName = strchr(typeSignature, ':');
+      typeSignatureLen = methodName - typeSignature;
+      methodName++;
+      methodNameLen = strlen(methodName);
+      }
+   else
+      {
+      methodName  = strchr(methodSignature, '.');
+      methodClassLen = methodName - methodClass;
+      methodName++;
+      typeSignature = strchr(methodName, '(');
+      typeSignatureLen = strlen(typeSignature);
```

Comment:
For my own clarification, are the consumers of `typeSignatureLen` expecting the first character to be `(`?

Reply:
I guess since the signature also contains the return type, it makes sense that `typeSignatureLen` starts at the `(`